src/main/java/org/jenkinsci/plugins/awsdevicefarm/AWSDeviceFarmRecorder.java

```diff
@@ -568,13 +583,17 @@ public void perform(@Nonnull hudson.model.Run<?, ?> build, @Nonnull FilePath wor
             DevicePool devicePool = adf.getDevicePool(project, devicePoolName);
 
             // Upload app.
+            Boolean isAndroidApp = false;
+            Boolean isIOSApp = false;
             String appArn = null;
             if (ifWebApp != null && ifWebApp){
                 writeToLog(log, "Tesing a Web App.");
 
             }
             else {
                 writeToLog(log, String.format("Using App '%s'", env.expand(appArtifact)));
+                isAndroidApp = appArtifact.toLowerCase().endsWith("apk") ? true : false;
+                isIOSApp = appArtifact.toLowerCase().endsWith("apk") ? true : false;
```

Comment:
`ipa` for the iOS app.

Reply:
Thanks for catching that @stefanverhoeff . Will change this.